model/team.go

```diff
@@ -97,7 +97,7 @@ func (o *Team) Etag() string {
 	return Etag(o.Id, o.UpdateAt)
 }
 
-func (o *Team) IsValid() *AppError {
```

Comment:
Why are we passing this as a property instead of just checking the configuration setting here?